src/components/icon/icon.test.jsx

```diff
@@ -11,80 +12,144 @@ describe('Icon', () => {
   });
 
   it('has title', () => {
-    const tree = renderer.create((
+    render((
       <Icon name="foobar" title="yo" />
-    )).toJSON();
-    expect(tree).toMatchSnapshot();
+    ));
+    expect(screen.getByRole('img')).toContainHTML('<title>yo</title>');
```

Comment:
```suggestion
    expect(screen.getByTitle('yo')).toExist();
```

Reply:
❗️ 

Let me see if it will work. I forgot about that particular query during development.